go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/golangci/golangci-lint v1.52.2
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
-	github.com/hewlettpackard/hpegl-metal-client v1.5.1
+	github.com/hewlettpackard/hpegl-metal-client v1.5.2-0.20230601202033-a64e83fff87b
```

Comment:
Will be updated once [PR 85](https://github.com/HewlettPackard/hpegl-metal-client/pull/85) is merged and new tag created,